test/e2e/specs/blocks/blocks__jetpack-forms.ts

```diff
@@ -30,6 +31,12 @@ const blockFlows: BlockFlow[] = [
 			],
 		}
 	),
+	new FormAiFlow( {
+		aiPrompt:
```

Comment:
**nit:non-blocking** - I think just "prompt" gets the point across and is more consistent with other AI blocks, but it's a very minor nit.